spec/requests/api/v1/passwords/create_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-describe 'POST api/v1/users/password' do
+describe 'POST api/v1/users/passwords' do
```

Comment:
I think the route is /password in singular. Check it by running `rails routes`

This is what I get
```
Prefix Verb       URI Pattern                             Controller#Action
POST              /api/v1/users/password(.:format)        api/v1/passwords#create
```